src/components/units-list/units-list-unit/units-list-unit.module.css

```diff
@@ -1,45 +1,51 @@
 .unit {
-  padding: 0 0 theme("spacing.2") theme("spacing.2");
+  padding: 6px 0;
+  padding-left: 14px;
+
+  &__selected {
+    border: 1px solid theme('colors.grey.dark');
```

Comment:
When we select unit and this border appears we can see little jumps of neighboor units. Maybe you can use a transparent border for default state of units to solve this problem

Reply:
yes, good point

Reply:
Decided to use outline